### ISSUE TITLE
Make it possible to work by having a global installation

### ISSUE THREAD

**Issue by jadjoubran** [jadjoubran/webdash#34]

**Comment by jadjoubran** [comment]
Behavior right now:
<img width="505" alt="screen shot 2018-03-27 at 2 30 56 pm" src="https://user-images.githubusercontent.com/2265232/37964828-86bf560e-31cb-11e8-818a-df714e907268.png">

app-root is returning the global node modules path, where it should look for where it's being run from

**Comment by jadjoubran** [comment]
This is now available for testing 🎉

if anyone's interested in testing it, please run the following and let me know if it works for you or not:

## Install it globally

```
npm install -g webdash@next
#run webdash serve in your project:
webdash serve
```

## Install it locally

```
npm install --save-dev webdash@next
npx webdash serve
```

**Comment by kakadiadarpan** [comment]
@jadjoubran What about the plugins, do we have to install them locally in our project?

**Comment by jadjoubran** [comment]
the config file should *always* be in the project.. I don't think it'll ever be possible to move it outside as all the values will change from 1 project to another (for example `manifestPath` will be different from project to another)

For the plugins, for now yes they have to be installed locally..
does it make sense to allow them to work globally?
Especially that they differ from project to project & if we look at other tools (like grunt & webpack) I think they require you to have the plugins installed locally

**Comment by kakadiadarpan** [comment]
Makes sense! 😄

**Comment by jadjoubran** [comment]
it seems this is broken because `is-global` is returning true when webdash is installed with `npm install webdash --save-dev` which is breaking the paths
Will be available in 1.4 instead

**Comment by jadjoubran** [comment]
Due to the nature of this feature, it must be tested with an npm prerelease:
available in version: `v1.4.0-2`

### Test instructions
## Local
`npm install --save-dev webdash@next`
`npx webdash serve`

## Global
`npm install -g webdash@next`
`webdash serve`
